systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -369,34 +394,78 @@ void testCreatingUsersWithSecretPrefix(ExtensionContext extensionContext) {
         assertNull(kubeClient(namespaceName).getSecret(scramShaSecret.getMetadata().getName()));
     }
 
-    synchronized void createBigAmountOfUsers(ExtensionContext extensionContext, String userName, String typeOfUser) {
-
-        int numberOfUsers = 100;
-
+    synchronized void createBigAmountOfUsers(ExtensionContext extensionContext, String namespace, String clusterName, String userName, String typeOfUser, int numberOfUsers) {
         for (int i = 0; i < numberOfUsers; i++) {
             String userNameWithSuffix = userName + "-" + i;
 
             if (typeOfUser.equals("TLS")) {
-                resourceManager.createResource(extensionContext, KafkaUserTemplates.tlsUser(userClusterName, userNameWithSuffix)
+                resourceManager.createResource(extensionContext, KafkaUserTemplates.tlsUser(clusterName, userNameWithSuffix)
                     .editMetadata()
-                        .withNamespace(NAMESPACE)
+                        .withNamespace(namespace)
                     .endMetadata()
                     .build());
             } else {
-                resourceManager.createResource(extensionContext, KafkaUserTemplates.scramShaUser(userClusterName, userNameWithSuffix)
+                resourceManager.createResource(extensionContext, KafkaUserTemplates.scramShaUser(clusterName, userNameWithSuffix)
                     .editMetadata()
-                        .withNamespace(NAMESPACE)
+                        .withNamespace(namespace)
                     .endMetadata()
                     .build());
             }
 
             LOGGER.info("Checking status of KafkaUser {}", userNameWithSuffix);
-            Condition kafkaCondition = KafkaUserResource.kafkaUserClient().inNamespace(NAMESPACE).withName(userNameWithSuffix).get()
+            Condition kafkaCondition = KafkaUserResource.kafkaUserClient().inNamespace(namespace).withName(userNameWithSuffix).get()
+                    .getStatus().getConditions().get(0);
+            LOGGER.info("KafkaUser condition status: {}", kafkaCondition.getStatus());
+            LOGGER.info("KafkaUser condition type: {}", kafkaCondition.getType());
+            assertThat(kafkaCondition.getType(), is(Ready.toString()));
+            LOGGER.info("KafkaUser {} is in desired state: {}", userNameWithSuffix, kafkaCondition.getType());
```

Comment:
I don't think that we want statuses of all users, created in this method, to be printed into `INFO` level.

Reply:
That's not my code, but I will change it to debug.

Reply:
Thanks